src/SFA.DAS.LevyTransferMatching/Interfaces/ILevyTransferMatchingService.cs

```diff
@@ -11,5 +11,6 @@ public interface ILevyTransferMatchingService
         Task<IEnumerable<Tag>> GetSectors();
         Task<IEnumerable<Tag>> GetJobRoles();
         Task<PledgeReference> CreatePledge(Pledge pledge);
+        Task<IEnumerable<Pledge>> GetPledges();
```

Comment:
I'm not sure that the Pledge class should be used here, given that it is also used in the CreatePledge method too. I don't like the idea of reusing classes. I suspect if we looked there'd be properties that go unused in one or other of the code paths due to the reuse. The classes used by each method should be defined in their own namespaces I'd have thought, be it api requests or api responses, as EI seems to do.